src/functions.php

```diff
@@ -477,3 +478,17 @@ function fix_exception(\Exception $exception): \Exception
 
     return $exception;
 }
+
+/**
+ * @return array<string, mixed>
+ */
+function load_dot_env(string $path = null): array
+{
+    $path ??= PathHelper::getRoot() . '/.env';
+
+    $dotenv = new Dotenv();
+    $dotenv->loadEnv($path);
+    unset($_ENV['SYMFONY_DOTENV_VARS']);
+
+    return $_ENV;
```

Comment:
Bah if $_ENV already contain data ?

Reply:
I went to the simple way of handling this, I tried with a raw script we have nothing in `$_ENV` so I don't think we should handle this case and let devs do it depending on their use-cases.

Reply:
okay